controllers/datafoundationclient.go

```diff
@@ -47,16 +50,29 @@ func dfcSetupWatches(controllerBuilder *builder.Builder) {
 		),
 	)
 	controllerBuilder.
-		Owns(&opv1a1.ClusterServiceVersion{}, csvPredicates).
-		Owns(&ocsclient.StorageClient{}).
-		Owns(&ocsclient.StorageClassClaim{})
+		Owns(&opv1a1.ClusterServiceVersion{}, csvPredicates)
+
+	if utils.MapCRDAvailability(storageClientCRD)[storageClientCRD] {
```

Comment:
Why are we running this here? You can read the same from the CRD map already provided to the reconcile